internal/log/markdown.go

```diff
@@ -0,0 +1,29 @@
+package log
+
+import (
+	"fmt"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/speakeasy-api/speakeasy/internal/charm/styles"
+	"regexp"
+)
+
+var (
+	patternToStyle = map[string]lipgloss.Style{
+		"\\*": lipgloss.NewStyle().Bold(true),
+		"_":   lipgloss.NewStyle().Italic(true),
+		"`":   styles.HeavilyEmphasized,
+	}
+)
+
+func StyleMarkdown(s string) string {
```

Comment:
Think this could do with a comment about what it does

Reply:
Ya you're right, it's a bit arcane